std/path.d

```diff
@@ -1270,7 +1270,7 @@ private auto _withExtension(R, C)(R path, C[] ext)
 {
     import std.algorithm.comparison : equal;
 
-    assert(testAliasedString!withExtension("file", "ext"));
+//    assert(testAliasedString!withExtension("file", "ext"));
```

Comment:
Move to a `version(DIP1000) {} else` unittest?

Reply:
Yes, if I don't find a general solution

Reply:
Looks like you didn't, so I suggest to go with `version(DIP1000) {} else` for now, s.t. we finally get entire Phobos working and then trim the small blocks out?